iac-extensions/arm/src/main/java/org/sonar/iac/arm/checkdsl/ContextualMap.java

```diff
@@ -50,6 +53,13 @@ public ContextualObject object(String name) {
       .orElse(ContextualObject.fromAbsent(ctx, name, this));
   }
 
+  public List<ContextualObject> objectsByPath(String path) {
+    return ArmTreeUtils.resolveProperties(path, tree).stream()
```

Comment:
As `tree` can be `null` in this context and the method supports it, I would suggest adding the annotation `@Nullable` on both `resolveProperties` methods.